zealot/fetcher/fetcher.ts

```diff
@@ -11,7 +11,7 @@ export type FetchArgs = {
   path: string
   method: string
   body?: string | FormData
-  headers?: Headers
+  headers?: any
```

Comment:
Can this be left as `Headers`? Or `object`

Reply:
Could be object, headers didn't work.

Reply:
Actually ended up needing to use Header, will change this back...